app/models/setting/auth.rb

```diff
@@ -20,7 +20,7 @@ def self.default_settings
       # websockets_encrypt depends on key/cert when true, so initialize it last
       self.set('websockets_encrypt', N_("VNC/SPICE websocket proxy console access encryption (websockets_ssl_key/cert setting required)"), !!SETTINGS[:require_ssl], N_('Websockets encryption')),
       self.set('login_delegation_logout_url', N_('Redirect your users to this url on logout (authorize_login_delegation should also be enabled)'), nil, N_('Login delegation logout URL')),
-      self.set('authorize_login_delegation_auth_source_user_autocreate', N_('Name of the external auth source where unknown externally authentication users (see authorize_login_delegation) should be created (keep unset to prevent the autocreation)'), nil, N_('Authorize login delegation auth source user autocreate')),
+      self.set('authorize_login_delegation_auth_source_user_autocreate', N_('Name of the external auth source where unknown externally authentication users (see authorize_login_delegation) should be created (If you want to prevent the autocreation, keep unset)'), 'External', N_('Authorize login delegation auth source user autocreate')),
```

Comment:
Can we even unset a value if there's a default? At least the help text doesn't match its function anymore since `keep` implies it's the default.

Reply:
@ekohl thanks for the correction in the message :) and yes, you can unset the value!